build/checkstyle/suppressions.xml

```diff
@@ -8,4 +8,6 @@
     <!-- Suppresses files in the src/main folder. To be used for rules which only apply to test
          code. -->
     <suppress files="[\\/]src[\\/]main[\\/].*|[\\/]generated-sources[\\/].*" id="TestScope" />
+    <!--  Suppress specific file  -->
+    <suppress files="hub/server/src/main/java/alluxio/hub/manager/process/ManagerProcessContext.java" checks="Indentation" />
```

Comment:
why is this file special?

Reply:
It has a ton of indent violations which I did have the heart to deal with 1 by 1. 😞

Reply:
ok we can leave this as an example and fix it later